frontend/src/components/Step3.module.css

```diff
@@ -0,0 +1,80 @@
+.red {
+  color: var(--color-ccidc-red);
+}
+
+.formContainer {
+  /* width: 100%;
+    margin: 3rem 10rem !important; */
+  background-color: var(--color-text-secondary);
+}
+
+.formSection {
+  margin: 3rem auto !important;
+  width: 80%;
+}
+
+.applyFacts {
+  width: 80%;
+  margin: 3rem auto !important;
+}
+.listText {
+  color: black;
+  margin: 0;
+}
+
+.bulletPoint {
+  list-style-type: disc;
+  color: var(--color-ccidc-red);
+}
+
+.warningText {
+  color: #ad281f;
+  margin: 3rem 10rem !important;
+}
+
+.red {
+  color: var(--color-ccidc-red);
+}
+
+.sectionTitle {
+  display: flex;
+  font-family: "Rubik";
+  font-size: 18px;
+  font-weight: 700;
+  margin-left: 100px;
+  margin-top: 0.25em;
+  margin-bottom: 30px;
+  padding-top: 2em;
+}
+
+.upload {
+  font-family: "Rubik";
+  font-size: 15px;
+  background-color: white;
+  color: black;
+  border: 1px solid black;
+  border-radius: 4px;
+  height: 48px;
+  width: 338px;
```

Comment:
Can we change this to relative units like em . This is pretty big for it to be hard coded.